moped-toolbox/random_project_seeds/utils/lookups.js

```diff
@@ -0,0 +1,63 @@
+const lookups = {
```

Comment:
it would not be not much effort to fetch these lookup values from the db - i punted on that but it would be a nice improvement.